bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -68,34 +68,17 @@ <h2><a class="mozilla" href="{{ url('mozorg.home') }}" title="{{ mozilla_logo_ti
       </div>
       <div class="header-container">
         <div class="header-content">
-          {% block main_content %}{% endblock %}
+          {% block main_content %}
+          {% block header_content %}{% endblock %}
```

Comment:
It seems like `main_content` and `header_content` could be separated rather than nested. 

I'm also not sure about the names... the whole top of the page seems to be the "header" so it feels like everything on the left is "header content" even though it was named `main_content` ... so now adding a new block for `header_content` could be confusing. Maybe instead of `header_content` we should call it `main_title` to pair with `main_content`? But I'm probably overthinking it.